platform/default/src/mbgl/storage/http_file_source.cpp

```diff
@@ -269,6 +269,12 @@ HTTPRequest::HTTPRequest(HTTPFileSource::Impl *context_, Resource resource_, Fil
       resource(std::move(resource_)),
       callback(std::move(callback_)),
       handle(context->getHandle()) {
+    if (resource.dataRange) {
```

Comment:
This needs to be implemented for the other HTTP file source implementations as well.

Reply:
I will provide this in the next commit.

Reply:
Done!